src/components/PatentList.tsx

```diff
@@ -84,19 +108,21 @@ function PatentList(props: any) {
   const [page] = React.useState(0)
   const [rowsPerPage] = React.useState(10)
   const [filterState, setFilterState] = React.useState({ searchTermFilter: '' })
+  const [currentDisplayInfo, setCurrentDisplayInfo] = React.useState('null');
```

Comment:
A string of 'null' isn't ideal, can we use nothing, actual null or an empty string?